### PR TITLE
Minor hunting bug fixes

### PR DESCRIPTION
**Pull request by danielsagi**
## Description
This PR fixes some typo issues and general misuses of function, which made some hunters fail silently.

## Contribution checklist
 - [x] I have read the Contributing Guidelines.
 - [ ] The commits refer to an active issue in the repository.
 - [ ] I have added automated testing to cover this case.

### CODE REVIEW

File: kube_hunter/modules/hunting/kubelet.py [hunk 1]
```diff
@@ -244,7 +244,7 @@ class DebugHandlers:
         """ all methods will return the handler name if successful """
 
         def __init__(self, path, pod, session=None):
-            self.path = path
+            self.path = path + "/" if not path.endswith("/") else ""
```
Comment (lizrice) [inline kube_hunter/modules/hunting/kubelet.py:247]: Hmm...I think this is going to leave path as an empty string if it does end with /
```
>>> p = "hello/"
>>> p = p + "/" if not p.endswith("/") else ""
>>> p
''
```

Some extra brackets will fix it
```
>>> p = "hello/"
>>> p = p + ("/" if not p.endswith("/") else "")
>>> p
'hello/'
```

File: kube_hunter/modules/hunting/etcd.py [hunk 2]
```diff
@@ -88,9 +88,7 @@ def db_keys_write_access(self):
         data = {"value": "remotely written data"}
         try:
             r = requests.post(
-                f"{self.protocol}://{self.event.host}:{ETCD_PORT}/v2/keys/message",
-                data=data,
-                timeout=config.network_timeout,
+                f"https://{self.event.host}:{ETCD_PORT}/v2/keys/message", data=data, timeout=config.network_timeout,
```
Comment (lizrice) [inline kube_hunter/modules/hunting/etcd.py:91]: Should this be `{self.event.protocol}`? I didn't check but I'm wondering if OpenPortEvent populates this. (In practice I would imagine any self-respecting deployment will use https anyway, so it's not a big deal, but just wondering about it)

File: kube_hunter/modules/hunting/kubelet.py (hunk 1 above)
Comment (danielsagi) [reply kube_hunter/modules/hunting/kubelet.py:247]: Youre right!

File: kube_hunter/modules/hunting/etcd.py (hunk 2 above)
Comment (danielsagi) [reply kube_hunter/modules/hunting/etcd.py]: @lizrice 
Open port event does not populate it... but I changed it to save the "https" in the event, in any case if we want to use it in other event later on.

**Pull request merged**